docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst

```diff
@@ -571,12 +571,28 @@ Specific outputs for specific models are specified in the following section.
     - Same as 9.2
     - 
   * - 11.0
-    - GBD 2021 update?
+    - GBD 2021 update
     - Baseline
-    - None
-    - 
+    - 400,000 population size per draw, 20 draws
+    - * Deaths, YLLs
+      * YLDs, stratified by pregnancy state
+      * Maternal disorder incident counts, stratified by anemia state
+      * Maternal hemorrhage incident counts, stratified by anemia state
+      * Anemia state person time, stratified by pregnancy state 
+      * Pregnancy state person time, stratified by birth outcome
+      * Pregnancy state transition counts, stratified by birth outcome
+      * Counts of birth outcomes
+      * Intervention counts
+    - Same as 9.2
+    - Should be the same as the adding locations test runs
+  * - 12.0
+    - GBD 2021 production runs
+    - Baseline
+    - Same as 9.2
+    - Same as 9.2 but including count of live births and still births 
```

Comment:
I think this observer might still have a bug? I don't remember if @hussain-jafari  already addressed it and we simply haven't run it yet or if it hasn't been fixed, but when we launched the previous production runs, this observer was counting birth outcomes at every timestep rather than just once, so we weren't able to use it in the emulator. Let's plan to fix it for the model 11 run if it hasn't already been fixed in the code and make sure it's working properly before launching model 12!

Reply:
I didn't write a fix for this so the code will have to be updated. @albrja feel free to reach out if you work on this I've taken a look at the observer